src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -859,7 +900,7 @@
         },
 
         "kind": {
-          "description": "The type of construct this logicalLocationComponent refers to. Should be one of 'function', 'member', 'module', 'namespace', 'package', 'resource', or 'type', if any of those accurately describe the construct.",
+          "description": "The type of construct this logicalLocationComponent refers to. Should be one of 'function', 'variable', 'member', 'module', 'namespace', 'package', 'parameter', 'resource', 'returnType', or 'type', if any of those accurately describe the construct.",
```

Comment:
Let's alphabetize these. I'll fix the copy in the OASIS repo.

Reply:
OK